scenes/cslib_string.txt

```diff
@@ -291,4 +291,39 @@
 	*set char_list_index char_list_len
 	*goto contains_loop
 *set cslib_ret false
-*return
\ No newline at end of file
+*return
+
+
+*comment CONSISTS_OF
+*comment ------------------------------
+*comment Checks if the given string consists
+*comment of any characters in a given list.
+*comment ------------------------------
+*comment 		params:
+*comment 			p_string(string): a string
```

Comment:
'a string' isn't helpful (that's already mentioned in the type annotation). It should read like documentation to the user. Something like "the string to check for the presence of the given characters".